test/common/schemaform/actions.unit.spec.js

```diff
@@ -193,4 +194,195 @@ describe('Schemaform actions:', () => {
       });
     });
   });
+  describe('uploadFile', () => {
+    let fetchMock;
+    let oldFetch;
```

Comment:
We pretty much never want to _actually_ use `fetch` in a unit test, right? Would it make sense to globalize this change and provide hooks to change what `fetch` returns?